doc/man1/mac.pod

```diff
@@ -0,0 +1,151 @@
+=pod
+
+=head1 NAME
+
+openssl-mac,
+mac - perform Message Authentication Code operations
+
+=head1 SYNOPSIS
+
+B<openssl mac>
+B<-name <mac_name>>
+[B<-help>]
+[B<-ctrl>]
+[B<-bin>]
+[B<-out filename>]
+[B<file>]
+
+B<openssl> I<mac> B<-name <mac_name>>  [B<...>]
+
+=head1 DESCRIPTION
+
+The message authentication code functions output the MAC of a supplied file
+in hexadecimal.
+
+=head1 OPTIONS
+
+=over 4
+
+=item B<-help>
+
+Print out a usage message.
+
+=item B<-name <mac_name>>
+
+Specifies name of a supported mac algorithm to be used. To see the list of
+supported mac's, use the command I<list -mac-algorithms>. This option must
+be specified.
+
+=item B<-bin>
+
+Output the MAC in binary form. Uses hexadecimal text format if not specified. 
+
+=item B<-out filename>
+
+Filename to output to, or standard output by default.
+
+=item B<-ctrl nm:v>
+
+Passes options to the MAC algorithm.
+The following options are supported:
+
+=over 4
+
+=item B<key:string>
+
+Specifies MAC key as alphanumeric string (use if key contain printable
+characters only). String length must conform to any restrictions of
+the MAC algorithm. A key must be specified for any MAC algorithm.
+
+=item B<hexkey:string>
+
+Specifies MAC key in hexadecimal form (two hex digits per byte).
+Key length must conform to any restrictions of the MAC algorithm.
+A key must be specified for any MAC algorithm.
+
+=item B<digest:string>
+
+Used by HMAC as alphanumeric string (use if key contain printable
+characters only). String length must conform to any restrictions of
+the MAC algorithm.
+To see the list of supported digests, use the command I<list -digest-commands>.
+
+=item B<cipher:string>
+
+Used by CMAC and GMAC to specifiy the cipher algorithm
+For CMAC it is one of AES-128-CBC, AES-192-CBC, AES-256-CBC, DES-EDE3-CBC.
+For GMAC it is one of AES-128-GCM, AES-256-GCM.
+
+=item B<iv:string>
+
+Used by GMAC to specify an IV as a alphanumeric string (use if IV contain
+printable characters only).
+
+=item B<hexiv:string>
+
+Used by GMAC to specify an IV in hexadecimal form (two hex digits per byte).
+
+
+=item B<outlen:int>
+
+Used by KMAC128 or KMAC256 to specify an output mac length. The default sizes are
```

Comment:
Should mac be capitalised here?  Or just dropped?

Reply:
dropped the mac word

Reply:
dropped it